app/controllers/admin/collections_controller.rb

```
@@ -15,8 +15,8 @@ def new
 
   def create
     @collection =
-      Collection.new_locked_ldp_object(permitted_attributes(Collection).merge(owner: current_user&.id))
-
+      Collection.new_locked_ldp_object(permitted_attributes(Collection)
+                .merge(owner: current_user&.id, community_id: @community.id))
```

I'm wondering if `current_user&.id` isn't just making it easier for future bugs to "work". Wouldn't it be better to crash if we are in a situation in which somehow a Collection is being created but there's no current user it's being created by?

The system's in an inconsistent state at that point AFAICT

---

Should be impossible to even get here if current_user is nil, since you being authorized twice to see if your an admin (in routes and controller). So definitely agree. Let it crash and crash hard if its here.

---

Done.